src/armature/Animation.ts

```diff
@@ -0,0 +1,176 @@
+import { Node } from './Node';
+import { Transformation } from './Transformation';
+
+import { remove } from 'lodash';
+
+type AnimationDescription = {
```

Comment:
A comment about what some of the fields mean would be helpful for me here as they are not immediately obvious. I'm specifically looking at:
- `curve`, what type of string is it expecting? should this be an enum?
- `lastTimeInCycle`, i have a vague idea about what this could do (is this internally used to keep track of the last cycle time)
- `lastCycle`, same as above, not 100% clear on what this does just by looking at it